src/Peachpie.CodeAnalysis/FlowAnalysis/AnalysisVisitor.cs

```diff
@@ -89,9 +89,14 @@ private void TraverseToBlock(object edgeLabel, FlowState/*!*/state, BoundBlock/*
                 // merge and check whether state changed
                 state = state.Merge(targetState);   // merge states into new one
 
-                if (state.Equals(targetState))
+                if (state.Equals(targetState) && !(target is ExitBlock))
```

Comment:
Special type condition for `ExitBlock` is just not nice.

Reply:
Replaced by ```BoundBlock.ForceRepeatedAnalysis``` in 529d61b.